src/utils/routingConstants.js

```diff
@@ -10,4 +10,5 @@ export const ROUTE_CURATIONS = '/curations'
 export const ROUTE_DISCORD = '/discord'
 export const ROUTE_STATS = '/stats'
 export const ROUTE_STATUS = '/status'
+export const ROUTE_FILES = '/files'
```

Comment:
I made a comment somewhere else about the name of the service route. Seems like we don't need a whole new route endpoint, rather just use the /attachments route. Effectively getting attachments with a fallback to use SWH. Just trying to keep the concept count down.

Reply:
@dabutvin thoughts on this?

Reply:
Actually, I was confused. This should be singular `/file` as well. The constant name should change too.